Release notes!

gboone opened the pull request.

Commit:
Release notes!

Commit:
Updates how long it's been since our last release

pburkholder commented on content/updates/2017-08-30-release-notes-plus-join-september-workshop.md:
```diff
@@ -0,0 +1,45 @@
+
+[**You’re invited to our free cloud.gov workshop on September 28th**](https://www.digitalgov.gov/event/hands-on-workshop-with-cloud-gov/). If you’re evaluating cloud.gov or getting started with it, you will learn how to launch applications and see the power and advantages that come from using a Platform as a Service. Whether you’re in-person or online, bring your laptop and prepare to learn how c ...
+
+If you use a cloud.gov account (not if you use EPA, FDIC, or GSA single sign-on login), [read our August 22 update about our plan to migrate your account’s authentication to OMB MAX](https://cloud.gov/updates/2017-08-21-max-gov-migration-instructions/). We’ll notify you at least three weeks before we make the switch. You can create your MAX account any time, and we encourage you to do it soon so  ...
```
I first read this as paragraph two of the workshop announcement. 

How about if each announcement gets its own slug, like:

```
**[cloud.gov Workshop](https://www.digitalgov.gov/event/hands-on-workshop-with-cloud-gov)**:  Lorem ipsum dolor ...

**cloud.gov authentication moves to MAX.gov:** Sed ut perspiciatis ...

fureigh replied:
^ I think that's a great idea.

fureigh commented on content/updates/2017-08-30-release-notes-plus-join-september-workshop.md:
```diff
@@ -0,0 +1,45 @@
+
+[**You’re invited to our free cloud.gov workshop on September 28th**](https://www.digitalgov.gov/event/hands-on-workshop-with-cloud-gov/). If you’re evaluating cloud.gov or getting started with it, you will learn how to launch applications and see the power and advantages that come from using a Platform as a Service. Whether you’re in-person or online, bring your laptop and prepare to learn how c ...
+
+If you use a cloud.gov account (not if you use EPA, FDIC, or GSA single sign-on login), [read our August 22 update about our plan to migrate your account’s authentication to OMB MAX](https://cloud.gov/updates/2017-08-21-max-gov-migration-instructions/). We’ll notify you at least three weeks before we make the switch. You can create your MAX account any time, and we encourage you to do it soon so  ...
```
I suggest changing `and` to `so` in `You can create your MAX account any time, and we encourage you to do it soon so you’re set well in advance.`

Commit:
Tidies up the release notes

pburkholder committed:
Update 2017-08-30-release-notes-plus-join-september-workshop.md

Comma goes before the "plus" and might as well be more specific about the date.

pburkholder commented on content/updates/2017-08-30-release-notes-plus-join-september-workshop.md:
```diff
@@ -0,0 +1,45 @@
+
+### Announcements
+
+[**cloud.gov Workshop:** You’re invited to our free cloud.gov workshop on September 28th](https://www.digitalgov.gov/event/hands-on-workshop-with-cloud-gov/). If you’re evaluating cloud.gov or getting started with it, you will learn how to launch applications and see the power and advantages that come from using a Platform as a Service. Whether you’re in-person or online, bring your laptop and pr ...
```
I'm going to nit-pick a bit:   Needs parallel structure or it reads like you'll "learn how to ... see the power and advantages"  Here's the wording I used in an email:

> Join us for this walk-through of how to release a cloud.gov web application, connect it to a database, and maintain the application’s security and compliance.  Participants will come away understanding how the cloud.gov Platform-as-a-Service model supports rapid development and continuous compliance better than other infrastructure approaches. 

Bring a laptop if you're online doesn't quite work. How about:

> Bring your laptop for the in-person experience, or boot up your workstation if you're remote, and prepare to learn how cloud.gov can work for you.

brittag reviewed (approved):
Approving since this looks good enough to ship; will leave merging to @gboone to decide if he wants additional adjustments.
